### PR TITLE
feat: [PIPE-18473]: NG Git Sync Health page

### PR DESCRIPTION
**Pull request by krishi0408**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by prasadsatamharness — DISMISSED** [review]

File: docs/platform/git-experience/git-sync-health-page.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+- In the **Webhooks** page, you can see the **Events** tab.
+
+![](./static/event_page.png)
+
+By defaut you will see the event that were **FAILED**, **PROCESSING**, **SKIPPED** or if there is a **WARNING**. You can modify the filter as per your requirement
+
+![](./static/event-page-gitsynchealth.png)
+
+In the event page you can see each event you have made to the files for the repo you have registered in webhook.
```
Comment (mohitg0795) [inline docs/platform/git-experience/git-sync-health-page.md:36]: this line is too confusing, lets try to simplify it.

File: docs/platform/git-experience/git-sync-health-page.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+![](./static/event_page.png)
+
+By defaut you will see the event that were **FAILED**, **PROCESSING**, **SKIPPED** or if there is a **WARNING**. You can modify the filter as per your requirement
+
+![](./static/event-page-gitsynchealth.png)
+
+In the event page you can see each event you have made to the files for the repo you have registered in webhook. 
+
+When you click on the button as pointed in the screenshot above you will see **Event Details** i.e file changes as well as payload that was sent.
```
Comment (mohitg0795) [inline docs/platform/git-experience/git-sync-health-page.md:38]: The screenshot above this line is unrelated

File: docs/platform/git-experience/git-sync-health-page.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+![](./static/event_details.png)
+
+In the **Files** you can see the the file that has been updates along with commit details.
```
Comment (mohitg0795) [inline docs/platform/git-experience/git-sync-health-page.md:42]: typo: "updated"

File: docs/platform/git-experience/git-sync-health-page.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+If an error occurs during sync, Harness marks the status as Failed and provides details.
```
Comment (mohitg0795) [inline docs/platform/git-experience/git-sync-health-page.md:44]: "If an error occurs during sync" -> This is not correct.

We should simply say that Harness validates all file changes and shows errors if there are any. Then we can provide the examples of different possible errors.
